com/edocti/jintro/Exercise6.java

```diff
@@ -0,0 +1,211 @@
+import java.lang.Math;
+
+public class Exercise6 {
+
+	// getName
+	public static String getName(String student) {
+		String[] lines = student.split("\n");

```

Comment:
A good thing is to always check your preconditions. In this case, you must assume that your array has 6 number of columns (or 7, not really sure)